eras/conway/impl/testlib/Test/Cardano/Ledger/Conway/Imp/UtxoSpec.hs

```diff
@@ -63,6 +79,38 @@ spec =
         [fromNativeScript spendingScript, fromNativeScript spendingScript]
           ++ extraScripts
           ++ extraScripts
+    describe "disjoint inputs and reference inputs" $ do
+      let
+        scriptHash lang = hashPlutusScript $ inputsOverlapsWithRefInputs lang
+        tx :: TxIn -> Tx era
+        tx txIn =
```

Comment:
We use `tx` binding in tests extensively for a test transaction. For consistency sake, I'd prefer to name a local function that does not conflict with that pattern and uses a slightly more descriptive name, eg:
```suggestion
        mkTestTx :: TxIn -> Tx era
        mkTestTx txIn =
```